GPU/GPUCommon.cpp

```diff
@@ -1075,6 +1081,22 @@ void GPUCommon::Execute_End(u32 op, u32 diff) {
 	}
 }
 
+void GPUCommon::Execute_TexScaleU(u32 op, u32 diff) {
+	gstate_c.uv.uScale = getFloat24(op);
+}
+
+void GPUCommon::Execute_TexScaleV(u32 op, u32 diff) {
+	gstate_c.uv.vScale = getFloat24(op);
+}
+
+void GPUCommon::Execute_TexOffsetU(u32 op, u32 diff) {
+	gstate_c.uv.uOff = getFloat24(op);
+}
+
+void GPUCommon::Execute_TexOffsetV(u32 op, u32 diff) {
+	gstate_c.uv.vOff = getFloat24(op);
```

Comment:
Yup, but in all other cases it works because we prescale UV for non-spline geometry these days, so the uvscaleoffset uniform isn't used, and I don't want to lose that optimization. So what we probably should do is to always dirty UVSCALEOFFSET when drawing splines/beziers, for a very small slowdown when drawing splines but still letting us combine draw calls in Tekken which is a massive speedup.